.github/PULL_REQUEST_PROCESS.md

```diff
@@ -21,6 +21,8 @@ The following items must all be true prior to a pull request being submitted to
 ## Code Review Process
 Community review of submitted pull requests is encouraged, and all pull requests must be reviewed by at least **two (2)** designated approvers before the change can be approved.  Once approved by two unique designated approvers, the change can be merged. An approver should NEVER merge their own change, even with two supporting approvals.  
 
+Then DNN Platform project strives to review and comment on incoming pull requests in a timely fashion, however, it may be necessary at times to hold incoming pull requests for a period of time during releases or other administrative processes.  
```

Comment:
```suggestion
The DNN Platform project strives to review and comment on incoming pull requests in a timely fashion, however, it may be necessary at times to hold incoming pull requests for a period of time during releases or other administrative processes.  
```